### PR TITLE
Setup Test Cases for the Showcase App

### PR DESCRIPTION
**Pull request by svdwyer**

### CODE REVIEW

File: showcase-app/test-cases.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+### 4. Switching Profiles Activates the Correct Profile
+
+- **Description**: Tests if switching between profiles correctly activates the correct profile in the app.
+- **Test Steps**:
+  - Setup 3 profiles in the extension.
+  - Switch between the profiles and verify that the correct profile is always activated on the main page.
+
+### 5. Removing Profile Inside the Extension Deactivates the Profile
+
+- **Description**: Tests if logging out inside the extension deactivates the profile.
```
Comment (pheyvaer) [inline showcase-app/test-cases.md:41]: The user can't log out of a profile considering that never log in with their WebID/IDP.

**Review by pheyvaer — CHANGES_REQUESTED** [review]

File: showcase-app/test-cases.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+### 1. Page Has Title And Headlines
+
+- **Description**: Tests if the main page has the correct title and header.
+- **Test Steps**:
+  - Open the main page.
+  - Check if the page title matches "Solid Auth Showcase".
+  - Check if the page main headline matches "COOL SOLID APP".
+  - Check if the page sub headline matches "Log in with your Solid Identity".
+
+### 2. Page Shows Error Message When Extension Is Not Available
```
[the remark this answers is not in the document]
Comment (svdwyer) [reply showcase-app/test-cases.md]: ✔️

File: showcase-app/test-cases.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+- **Description**: Tests if an error message is displayed when no connection can be made to the Solid Identity Extension.
```
[the remark this answers is not in the document]
Comment (svdwyer) [reply showcase-app/test-cases.md:18]: ✔️

File: showcase-app/test-cases.md (hunk 1 above)
Comment (svdwyer) [reply showcase-app/test-cases.md]: I've reworded it to "remove profile" instead of logging out

**Pull request merged**